css/terminal7.css

```diff
@@ -767,11 +749,20 @@ audio { display: none; }
     text-align: left;
     padding-left: 1.5em;
     text-shadow: none;
+    z-index: 20;
+    border-bottom: 1px rgba(255, 255, 255, 0.7) solid;
```

Comment:
Let's use the current border color of we use the separate buttons on the bnav bar: rgba(55, 55, 2, 1)

Reply:
Will change it
the root color is --dark-border